parser/indexedfile.go

```diff
@@ -127,7 +127,7 @@ func indexFiles(files []string, indexingThreads int, res *resources.Resources) [
 					res.Log.WithFields(log.Fields{
 						"file":  files[j],
 						"error": err.Error(),
-					}).Warning("An error was encountered while indexing a file")
+					}).Debug("An error was encountered while indexing a file")
```

Comment:
These log messages cluttered up the log files and we've never had much use for them. By switching this to debug it will still get logged to the database log collection but not put in the log files by default.